packages/ts-codegen/src/__tests__/index.test.ts

```diff
@@ -1975,6 +1975,55 @@ describe(generateTS, () => {
       "
     `);
   });
+
+  it("legacy no models", () => {
```

Comment:
It is no English sentence 🙃 

```suggestion
  it("does not generate models in legacy mode", () => {
```